src/rest/Facet/FacetRequest.ts

```diff
@@ -1,5 +1,12 @@
 import { FacetValueState } from './FacetValueState';
 import { FacetSortCriteria } from './FacetSortCriteria';
+import { IRangeValue } from '../RangeValue';
+
+export enum FacetType {
```

Comment:
Documentation necessary for this?

Reply:
Since we are linking to it on line 53, what would be display currently vs. if we documented it?